siac/walletcmd.go

```diff
@@ -192,8 +192,11 @@ func walletinitcmd() {
 }
 
 // walletload033xcmd loads a v0.3.3.x wallet into the current wallet.
-func walletload033xcmd(source string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+func walletload033xcmd() {
+	fmt.Print("Enter 033x Location: ")
+	var source string
+	fmt.Scanln(&source)
```

Comment:
The rationale for doing it this way was to match the way we load seeds, ie interactively.  I guess it doesn't really make sense since the 033x and siag loaders take a file location, not the actual data.